### DISCUSSION TITLE
Can be users from Authentik provisioned and synced to external LDAP ?

### DISCUSSION THREAD

**Discussion by czslop01** [goauthentik/authentik#10227] [Q&A]
I'm testing usability of authentik product in my project. One part is about to provision and  synchronize users from Authentik to external LDAP - 389-ds-base. I found possibility of synchronization from external LDAP to authentik, but not sure how is it in opposite. 
Authentication is then directly against external LDAP.

**Comment by PentaPaetzold** [comment]
Giving this a vote up. I have set up LDAP Source and see ldap_sync running. The users of the LDAP are stored in the given path. But if i change the path of one user top show up in LDAPs slug, its not synced to the LDAP- Server.
How to get the Authentik- Users back to LDAP? Maybe some way th get LDIF out of authentik for Export?

**Comment by Mase3206** [comment]
I know it's not exactly what we're looking for, but it looks like the FreeIPA team made a SCIM to AD, FreeIPA, and 389ds bridge. It seems promising, as long as you're willing to use it (with SCIM) an intermediary. https://github.com/freeipa/ipa-tuura/tree/main
